gatorgrade/output/output_tools.py

```diff
@@ -1,3 +1,7 @@
 # This file is used for storing supporting functions and tools that are used by the methods in output_functions.
 # The main use of this class is to de-clutter output_functions.py for better readability.
-# One example of a function to be written here is the splitting of strings retrieved from the input team, at spaces, into lists.
\ No newline at end of file
+# One example of a function to be written here is the splitting of strings retrieved from the input team, at spaces, into lists.
+
+def get_simple_command_string(command):

```

Comment:
Please add type hints to your parameters to make it more in line with best programming practices.